build-logic/convention/src/main/kotlin/com/google/samples/apps/nowinandroid/AndroidCompose.kt

```diff
@@ -40,16 +44,18 @@ internal fun Project.configureAndroidCompose(
             kotlinCompilerExtensionVersion = libs.findVersion("androidxComposeCompiler").get().toString()
         }
 
-        kotlinOptions {
-            freeCompilerArgs = freeCompilerArgs + buildComposeMetricsParameters()
-        }
-
         dependencies {
             val bom = libs.findLibrary("androidx-compose-bom").get()
             add("implementation", platform(bom))
             add("androidTestImplementation", platform(bom))
         }
     }
+
+    tasks.withType<KotlinCompile>().configureEach {
```

Comment:
Is setting freeCompilerArgs on tasks different from older way of setting it through kotlinOptions extension?

Reply:
In the interest of getting this merged I will resolve this conversation, but have passed this on to the tools team. Hopefully they can respond.

Reply:
nit: you don't even need to call `.configureEach`.
The following code does the same thing 🙂 

```kotlin
    tasks.withType<KotlinCompile> {
        /* */
    }
```